src/js/common/schemaform/reducers/index.js

```diff
@@ -104,6 +104,7 @@ export default function createSchemaFormReducer(formConfig) {
       loadedStatus: LOAD_STATUSES.notAttempted,
       version: formConfig.version,
       formId: formConfig.formId,
+      lastSaveDate: null,
```

Comment:
@cvalarida Does this make sense? Or should I update this in the metadata for loadedData? We won't always have data in loadedData after a save, though.